develop-docs/sdk/expected-features/spotlight.mdx

```diff
@@ -0,0 +1,414 @@
+---
+title: Spotlight
+sidebar_order: 5
+---
+
+[Sentry Spotlight](https://spotlightjs.com/) is a local development tool that provides real-time observability for errors, traces, logs, and performance data during development. SDKs should implement support for Spotlight to unlock the power of Sentry for local development.
+
+## Configuration Options
+
+SDKs should support Spotlight configuration through one of two approaches:
```

Comment:
This document already uses uppercase for a couple of occurrences or "MUST", "SHOULD", and "MAY". So it seems like we're following: https://www.ietf.org/rfc/rfc2119.txt. Therefire, I would write all occurrences uppercase
```suggestion
SDKs SHOULD support Spotlight configuration through one of two approaches:
```